File: pytools/examples/golang/src/server/proto/req_scene_req_players.go
package proto

import (
	"packet"
)

type ReqSceneReqPlayers struct {
}

func ReqSceneReqPlayersDecode(pack *packet.Packet) *ReqSceneReqPlayers {
	reqSceneReqPlayers := &ReqSceneReqPlayers{}

	return reqSceneReqPlayers
}

File: pytools/data/data.code.lua.server/ReqSceneReqPlayers.lua
module("ReqSceneReqPlayers", package.seeall)


function new()
	local tab_obj = {}
	setmetatable(tab_obj, {__index = ReqSceneReqPlayers})
	tab_obj:init()
	return tab_obj
end

function init(self)
end


function decode(self, pack)
	return self
end


